src/components/NewLocationPage/ChartFooter/utils.ts

```diff
@@ -31,3 +47,20 @@ export interface DialogProps {
   modalContent: MetricModalContent;
   modalHeader: string;
 }
+
+function formattedHsaCovidUsage(
+  metric: ExploreMetric,
+  projections: Projections,
+) {
+  /* Grab and format the corresponding HSA-level hospital metric data. */
+  let hsaValue;
+  if (metric === ExploreMetric.ICU_HOSPITALIZATIONS) {
+    hsaValue = projections.primary.currentHsaIcuInfo.currentUsageCovid;
+  } else if (metric === ExploreMetric.HOSPITALIZATIONS) {
+    hsaValue = projections.primary.currentHsaHospitalInfo.currentUsageCovid;
+  } else {
+    fail('Unexpected HSA level metric.');
+  }
+
+  return formatValue(Metric.ADMISSIONS_PER_100K, hsaValue, '-');
```

Comment:
Why is this ADMISSIONS_PER_100K?

Reply:
Ah, Since hospitalizations and ICU hospitalizations don't exist in the `Metric` enum, I chose a similar metric that did exist. (This function formats the numbers based on the mapping below). https://github.com/covid-projections/covid-projections/blob/e16886aa1624de00b3b0d292c99cfbe69e5cf103/src/common/metric.tsx#L112

This is kind of hacky as it's not 1-1 with the actual metrics--should I look to maybe create matching metrics, or extend formatValue to accept `ExploreMetrics` as well as `Metrics`?

Reply:
Looking at the code, I think [this](https://github.com/covid-projections/covid-projections/blob/00994c38a508fe8b496608787de11e17a57bd883/src/components/Charts/Groupings.tsx#L225) is the code that ends up formatting the normal value for the footer.  So I think I'd just do `formatDecimal(hsaValue, 1)` here.

Reply:
Oh perfect, that's much more simple--thanks